source/RunClient.py

```diff
@@ -18,14 +17,19 @@ def RunClient():
     connection.DoConnect((host, int(port)))
     clientState = ClientState(ruleset)
     gameControl = Controller(clientState)
-    handView = HandView(gameControl)
-    tableView = TableView()
-    while 1:
+    createDisplay = CreateDisplay(gameControl)
+    handView = HandView(gameControl, createDisplay.display)
+    tableView = TableView(createDisplay.display)
+    while True:
+        createDisplay.refresh()
         handView.nextEvent()
         connection.Pump()
         gameControl.Pump()
         tableView.Pump()
-        handView.render()
+        handView.update()
+        tableView.playerByPlayer()
+        note = gameControl.note
```

Comment:
Having this blip at a rate controlled by the game loop and not by the handView itself is limiting. If a event triggers a notification we can't show it until after we leave the handView nextEvent method and if an event triggers several notes in a row I think only the last one is seen.

Reply:
After discussion we concluded it would be best for some events to give a fresh note, while other events might append onto the existing note.  Some events might have multiple appends.